### PR TITLE
[bitnami/logstash] Adding tls-secrets.yaml #4768

### LINKED ISSUE #4768: [logstash] Missing tls-secrets.yaml templates
**Which chart**:
logstash 2.0.0

**Describe the bug**
Secrets for TLS are not created because tls-secrets.yaml templates are missing.

### PR DESCRIPTION
**Pull request by ikus060**
**Description of the change**

Adding tls-secrets.yml to logstash chart

**Benefits**

?

**Possible drawbacks**

unknown

**Applicable issues**

  - fixes #4768

**Additional information**

NA

**Checklist** 
- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Variables are documented in the README.md
- [X] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)
- [X] If the chart contains a `values-production.yaml` apart from `values.yaml`, ensure that you implement the changes in both files

### CODE REVIEW

File: bitnami/logstash/templates/tls-secret.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+  {{- end }}
+type: kubernetes.io/tls
+data:
+  tls.crt: {{ .certificate | b64enc }}
+  tls.key: {{ .key | b64enc }}
+---
+{{- end }}
+{{- else if and .Values.ingress.tls (not .Values.ingress.certManager) }}
+{{- $ca := genCA "aspnet-core-ca" 365 }}
```
Comment (FraPazGal) [inline bitnami/logstash/templates/tls-secret.yaml:22]:
```suggestion
{{- $ca := genCA "logstash-ca" 365 }}
```

**Review by FraPazGal — CHANGES_REQUESTED** [review]
Thanks for the PR @ikus060! Could you please take a look at my comment?

File: bitnami/logstash/templates/tls-secret.yaml (hunk 1 above)
Comment (ikus060) [reply bitnami/logstash/templates/tls-secret.yaml]: Fixed

**Review by FraPazGal — APPROVED** [review]
LGTM!

**Pull request merged**